CioFirebaseWrapper.podspec

```
@@ -0,0 +1,28 @@
+Pod::Spec.new do |spec|
+  spec.name         = "CioFirebaseWrapper"
+  spec.version      = "1.0.0"
+  spec.summary      = "Customer.io Firebase Wrapper SDK for iOS."
+  spec.homepage     = "https://github.com/customerio/customerio-ios-fcm"
+  spec.documentation_url = 'https://customer.io/docs/sdk/ios/'
+  spec.changelog    = "https://github.com/customerio/customerio-ios-fcm/blob/#{spec.version.to_s}/CHANGELOG.md"
+  spec.license      = { :type => "MIT", :file => "LICENSE" }
+  spec.author       = { "CustomerIO Team" => "win@customer.io" }
+  spec.source       = { :git => 'https://github.com/customerio/customerio-ios-fcm.git', :tag => spec.version.to_s }
+
+  spec.swift_version = '5.5'
+  spec.cocoapods_version = '>= 1.11.0'
+
+  spec.platform = :ios
+  spec.ios.deployment_target = "15.0"
+
+  path_to_source_for_module = "Sources"
+  spec.source_files = "#{path_to_source_for_module}/**/*{.swift}"
+  
+  spec.module_name = "CioFirebaseWrapper"
+  
+  # Add main SDK dependency
+  spec.dependency "CustomerIOMessagingPushFCM", ">= 3.13.0"
+  
+  # Add Firebase dependency - major version 12 up to next major version
+  spec.dependency "FirebaseMessaging", "~> 12.0"
```

Shouldn't this be same range as before?

```suggestion
  spec.dependency "FirebaseMessaging", ">= 8.7.0", "< 13.0.0"
```

---

This raises the min deployment to 15 which is supported by Firebase v12, allowing lower versions will cause the resolution issue

---

Not sure I fully understand how this causes resolution issues. My understanding was that if we keep the same range, customers could still use Firebase versions like `10` or `11`, the only requirement would be updating the minimum iOS deployment target to `15` for them, not necessarily using Firebase `12`. Isn't that correct?

---

Done